Plugins/BridgeJS/Tests/BridgeJSToolTests/__Snapshots__/ExportSwiftTests/Namespaces.swift

```diff
@@ -0,0 +1,126 @@
+// NOTICE: This is auto-generated code by BridgeJS from JavaScriptKit,
+// DO NOT EDIT.
+//
+// To update this file, just rebuild your project or run
+// `swift package bridge-js`.
+
+@_spi(BridgeJS) import JavaScriptKit
+
+@_expose(wasm, "bjs_create")
+@_cdecl("bjs_create")
+public func _bjs_create() -> Void {
+    #if arch(wasm32)
```

Comment:
if this is just following an existing pattern, that is fine. But figured I'd mention it's more common to use `#if os(WASI)` than `#if arch(wasm32)`. Not sure how much it matters. They should both do the same thing.

Reply:
Yes, using their existing pattern is fine